Tests/iaas/scs-0104-v1.1-images.yaml

```diff
@@ -0,0 +1,30 @@
+images:
+- name: "Ubuntu 24.04"
+  source:
+  - https://cloud-images.ubuntu.com/releases/noble/
+  - https://cloud-images.ubuntu.com/noble/
+  status: mandatory
+- name: "Ubuntu 22.04"
+  source:
+  - https://cloud-images.ubuntu.com/releases/jammy/
+  - https://cloud-images.ubuntu.com/jammy/
+- name: "ubuntu-capi-image"
+  name_scheme: "ubuntu-capi-image v[0-9]\\.[0-9]+(\\.[0-9]+)?"
+  source: https://swift.services.a.regiocloud.tech/swift/v1/AUTH_b182637428444b9aa302bb8d5a5a418c/openstack-k8s-capi-images/ubuntu-2204-kube
+  status: recommended
+- name: "Ubuntu 20.04"
```

Comment:
While touching this list, should we remove this?

Reply:
@berendt I tend to to agree. However: the status quo of the standard and check is as follows: without a `status`, the conformance check will not complain at all if this image is missing. However, it will complain if it is available but from the wrong source.

What we probably want, and what we discussed internally: there needs to be a way phase out old OS images, i.e., new `status` values in the YAML. @mbuechse proposed `discouraged` and `forbidden` (the counterparts for `recommended` and `mandatory`). It needs to be discussed, whether such an image should be removed completely, or visibility changed to `community` or something.

Reply:
To be discussed as part #684

Reply:
Two questions here:

1. does it hurt to specify the preferred source url for a distro that is EOL? Maybe not.
2. shall we discourage CSPs from providing these EOL distros? Maybe yes.

So yeah, in the end, maybe keep them in the file, but go ahead with #684 and mark them "discouraged".